packages/typography/src/styled/StyledFont.tsx

```diff
@@ -29,43 +29,46 @@ export const THEME_SIZES: Record<TypographySize, ThemeSize> = {
   '3xlarge': 'xxxl'
 };
 
-const fontStyles = (props: IStyledFontProps & ThemeProps<DefaultTheme>) => {
-  const monospace =
-    props.isMonospace && ['inherit', 'small', 'medium', 'large'].indexOf(props.size!) !== -1;
-  const fontFamily = monospace && props.theme.fonts.mono;
-  const direction = props.theme.rtl ? 'rtl' : 'ltr';
+const fontStyles = ({
+  hue,
+  isBold,
+  isMonospace,
+  size,
+  theme
+}: IStyledFontProps & ThemeProps<DefaultTheme>) => {
+  const monospace = isMonospace && ['inherit', 'small', 'medium', 'large'].indexOf(size!) !== -1;
+  const fontFamily = monospace && theme.fonts.mono;
+  const direction = theme.rtl ? 'rtl' : 'ltr';
   let fontSize;
   let fontWeight;
   let lineHeight;
-  let color;
+  let color = getColor({ theme, variable: 'foreground.default' });
 
   if (monospace) {
-    if (props.size === 'inherit') {
+    if (size === 'inherit') {
       fontSize = 'calc(1em - 1px)';
       lineHeight = 'normal';
     } else {
-      const themeSize = THEME_SIZES[props.size!];
+      const themeSize = THEME_SIZES[size!];
 
-      fontSize = math(`${props.theme.fontSizes[themeSize]} - 1px`);
-      lineHeight = math(`${props.theme.lineHeights[themeSize]} - 1px`);
+      fontSize = math(`${theme.fontSizes[themeSize]} - 1px`);
+      lineHeight = math(`${theme.lineHeights[themeSize]} - 1px`);
     }
-  } else if (props.size !== 'inherit') {
-    const themeSize = THEME_SIZES[props.size!];
+  } else if (size !== 'inherit') {
+    const themeSize = THEME_SIZES[size!];
 
-    fontSize = props.theme.fontSizes[themeSize];
-    lineHeight = props.theme.lineHeights[themeSize];
+    fontSize = theme.fontSizes[themeSize];
+    lineHeight = theme.lineHeights[themeSize];
   }
 
-  if (props.isBold === true) {
-    fontWeight = props.theme.fontWeights.semibold;
-  } else if (props.isBold === false || props.size !== 'inherit') {
-    fontWeight = props.theme.fontWeights.regular;
+  if (isBold === true) {
+    fontWeight = theme.fontWeights.semibold;
+  } else if (isBold === false || size !== 'inherit') {
+    fontWeight = theme.fontWeights.regular;
   }
 
-  if (props.hue) {
-    const shade = props.hue === 'yellow' ? 700 : 600;
```

Comment:
No longer needed since the default shade for all hues is now `700` in `light` mode.